### PR TITLE
PN-90: Genotypic seeding results need to be combined with phenotypic seeding.

### PR DESCRIPTION
**Pull request by veronikaslc**
Added genes to indexing of the Patient.

### CODE REVIEW

**Comment by veronikaslc** [comment]
@allasm @mjshepherd, please review

**Comment by veronikaslc** [comment]
@allasm @mjshepherd please review

File: resources/solr-configuration/src/main/resources/patients/conf/schema.xml [hunk 1]
```diff
@@ -61,6 +61,10 @@
     <field name="reporter" type="string" indexed="false" stored="true" required="true" />
     <field name="visibility" type="string" indexed="true" stored="true" required="false" />
     <field name="accessLevel" type="int" indexed="true" stored="true" required="false" />
+    <field name="phenotype_ancestors" type="string" indexed="true" stored="true" multiValued="true"/>
+	<field name="solved_genes" type="string" indexed="true" stored="true" multiValued="true"/>
+	<field name="candidate_genes" type="string" indexed="true" stored="true" multiValued="true"/>
+	<field name="rejected_genes" type="string" indexed="true" stored="true" multiValued="true"/>
```
Comment (mjshepherd) [inline resources/solr-configuration/src/main/resources/patients/conf/schema.xml]: indent

**Comment by mjshepherd** [comment]
+1 Code looks good. But does this fix the issue? It seems to only add indexing to genes and all ancestors.

**Comment by veronikaslc** [comment]
This is afix for the issue from the side of phenotips code, the actual code that fixes the issue is in the second pull request https://github.com/phenotips/patient-network/pull/93

**Comment by buske** [comment]
Replaced in favor of #1914.

**Pull request closed without merging**